src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -472,6 +472,11 @@
       "datatype": "bool",
       "value": "(markup == 'xaml')"
     },
+    "useToolkit": {
```

Comment:
This is not needed. `useMaterial` already does this check

Reply:
With these changes we can remove the preset check for useMaterial, which will mean Material can be added to Blank template